bin/redshift/run_mvn_tests_on_aws.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Runs Redshift connector tests.
+#
+# Run setup_aws_redshift.sh as a prerequisite for creating the Redshift instance and viewing
+# required environment variables.
+#
+# Usage:
+# run_mvn_tests_on_aws.sh
+
+set -xeuo pipefail
+
+ROOT_DIR="$(git rev-parse --show-toplevel)"
+cd "$ROOT_DIR"
+
+trap terminate INT TERM EXIT
+
+suite_exit_code=0
+
+MAVEN_OPTS="$MAVEN_INSTALL_OPTS" ./mvnw -s .github/settings.xml\
+    test \
+    -B -Dair.check.skip-all=true -Dmaven.javadoc.skip=true --fail-at-end \
+    -Dtest.redshift.jdbc.user="${REDSHIFT_USER}" \
+    -Dtest.redshift.jdbc.password="${REDSHIFT_PASSWORD}" \
+    -Dtest.redshift.jdbc.endpoint="${REDSHIFT_ENDPOINT}/" \
```

Comment:
that means the endpoint is not an endpoint, but just a prefix of it

can we fix REDSHIFT_ENDPOINT definition so that it includes all the necessary trailing slashes?

Reply:
Do you mean to provide it like this `"jdbc:redshift://${REDSHIFT_ENDPOINT}:${REDSHIFT_PORT}/testdb" in the `test.redshift.jdbc.endpoint` / ``test.redshift.jdbc.url` ?